src/main/java/org/sonatype/nexus/repository/composer/internal/ComposerHostedUploadHandler.java

```diff
@@ -37,14 +37,17 @@
   @Nonnull
   @Override
   public Response handle(@Nonnull final Context context) throws Exception {
-    String path = ComposerPathUtils.buildZipballPath(context);
+    String vendor = ComposerPathUtils.getVendorToken(context);
```

Comment:
Could statically import here. I dont think it looses context

Reply:
> Could statically import here. I dont think it looses context

I just don't like static imports, but since we use them all over NXRM, I'm glad you pointed it out. :) There's something to be said for consistency within a codebase (or related codebases).

Static imports added in https://github.com/sonatype-nexus-community/nexus-repository-composer/pull/18/commits/66f75d484c0e4cbb9bfed661ea6e41b97312916b.